DiscordChatExporter.Core/Exporting/Writers/Html/PreambleTemplate.cshtml

```diff
@@ -779,13 +794,30 @@
-            <path fill="#f4f5fb" d="M530,215a25,25,0,0,1-25-25V50a25,25,0,0,1,16.23-23.41L693.41,198.77A25,25,0,0,1,670,215Z"/>
-            <path fill="#7789c4" d="M530,70.71,649.29,190H530V70.71M530,0a50,50,0,0,0-50,50V190a50,50,0,0,0,50,50H670a50,50,0,0,0,50-50Z"/>
-        </symbol>
+    <svg style="display: none" xmlns="http://www.w3.org/2000/svg">
+        <defs>
+            <symbol id="attachment-icon" viewBox="0 0 720 960">
+                <path fill="#f4f5fb" d="M50,935a25,25,0,0,1-25-25V50A25,25,0,0,1,50,25H519.6L695,201.32V910a25,25,0,0,1-25,25Z"/>
+                <path fill="#7789c4" d="M509.21,50,670,211.63V910H50V50H509.21M530,0H50A50,50,0,0,0,0,50V910a50,50,0,0,0,50,50H670a50,50,0,0,0,50-50h0V191Z"/>
+                <path fill="#f4f5fb" d="M530,215a25,25,0,0,1-25-25V50a25,25,0,0,1,16.23-23.41L693.41,198.77A25,25,0,0,1,670,215Z"/>
+                <path fill="#7789c4" d="M530,70.71,649.29,190H530V70.71M530,0a50,50,0,0,0-50,50V190a50,50,0,0,0,50,50H670a50,50,0,0,0,50-50Z"/>
+            </symbol>
+            <symbol id="icon-channelpinnedmessage" fill="none" height="17" viewBox="0 0 17 17" width="17">
+                <title>ChannelPinnedMessage</title>
+                <path d="m16.908 8.39684-8.29587-8.295827-1.18584 1.184157 1.18584 1.18584-4.14834 4.1475v.00167l-1.18583-1.18583-1.185 1.18583 3.55583 3.55502-4.740831 4.74 1.185001 1.185 4.74083-4.74 3.55581 3.555 1.185-1.185-1.185-1.185 4.1475-4.14836h.0009l1.185 1.185z" fill="#b9bbbe"/>
+            </symbol>
+            <symbol id="icon-call" width="18" height="18">
+                <title>Call</title>
+                <path fill="#3ba55c" fill-rule="evenodd" d="M17.7163041 15.36645368c-.0190957.02699568-1.9039523 2.6680735-2.9957762 2.63320406-3.0676659-.09785935-6.6733809-3.07188394-9.15694343-5.548738C3.08002193 9.9740657.09772497 6.3791404 0 3.3061316v-.024746C0 2.2060575 2.61386252.3152347 2.64082114.2972376c.7110335-.4971705 1.4917101-.3149497 1.80959713.1372281.19320342.2744561 2.19712724 3.2811005 2.42290565 3.6489167.09884826.1608492.14714912.3554431.14714912.5702838 0 .2744561-.07975258.5770327-.23701117.8751101-.1527655.2902036-.65262318 1.1664385-.89862055 1.594995.2673396.3768148.94804468 1.26429792 2.351016 2.66357424 1.39173858 1.39027775 2.28923588 2.07641807 2.67002628 2.34187563.4302146-.2452108 1.3086162-.74238132 1.5972981-.89423205.5447887-.28682915 1.0907006-.31944893 1.4568885-.08661115.3459689.2182151 3.3383754 2.21027167 3.6225641 2.41611376.2695862.19234426.4144887.5399137.4144887.91672846 0 .2969525-.089862.61190215-.2808189.88523346"/>
+            </symbol>
+            <symbol id="icon-guildmemberjoin" height="18" width="18">
+                <title>GuildMemberJoin</title>
+                <g fill="none" fill-rule="evenodd">
+                    <path d="m18 0h-18v18h18z"/>
+                    <path d="m0 8h14.2l-3.6-3.6 1.4-1.4 6 6-6 6-1.4-1.4 3.6-3.6h-14.2" fill="#3ba55c"/>
+                </g>
+            </symbol>
+        </defs>
```

Comment:
Also can we make these consistent, i.e.:

- Follow the original naming convention of `xxx-icon` instead of `icon-xxx`
- Use `viewbox="0 0 <width> <height>"` instead of `width="..." height="..."`

Reply:
Changing name will create an issue with the <svg><use href=></use></svg> reference in the messagegrouptemplate.cshtml since the svg is referenced by the id which is "icon-" + message.Kind.ToString().ToLowerInvariant().

`<use href="#@message.Kind.ToString().ToLowerInvariant()-icon"` is this valid code?


I will adjust the viewbox - that's a good idea.

Reply:
Yes. Maybe you can make an extension method such that converts `ChannelPinnedMessage` to `Channel-Pinned-Message` (and then lowercase it). You can use RegEx to replace a pair of a lowercase character and an upper case character, with a dash in-between.

Reply:
Please check how it looks now. Is this okay?

Reply:
Yeah, this is great, thanks!